tests/tests_unit/test_api_client.py

```diff
@@ -903,12 +903,12 @@ class TestHelpers:
         "input, emulator_url, expected",
         [
             (
-                "http://localtest.com/api/1.0/projects/test-project/analytics/models",
+                "http://localtest.com/api/1.0/projects/test-project/modelhosting/models",
```

Comment:
We're mixing 1.0 (which I guess is Nostromo 1.0, not CDF v1) and 0.1 here, looks a bit odd?